### PR TITLE
translate: Add Russian translation

### PR DESCRIPTION
**Pull request by fisuri**

### CODE REVIEW

**Comment by fisuri** [comment]
[patch](https://raw.githubusercontent.com/fisuri/wallpaper-selector/main/translate%3A%20Add%20Russian%20translation.patch)

File: po/ru.po [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# Italian translation for io.github.davidoc26.wallpaper_selector.
```
Comment (Davidoc26) [inline po/ru.po:1]: Need correction

File: po/ru.po [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+# Malykh Yaroslav <fisuri@murena.io>, 2024.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: io.github.davidoc26.wallpaper_selector\n"
+"Report-Msgid-Bugs-To: https://github.com/davidoc26/wallpaper-selector\n"
+"POT-Creation-Date: 2022-10-23 16:53+0300\n"
+"PO-Revision-Date: 2023-03-03 20:11+0100\n"
+"Last-Translator: Albano Battistella <albanobattistella@gmail.com>\n"
+"Language-Team: Italian <>\n"
```
Comment (Davidoc26) [inline po/ru.po:14]: Need correction here (and lines 13, 14, 15)

**Review by Davidoc26 — CHANGES_REQUESTED** [review]
Corrections needed

**Review by Davidoc26 — APPROVED** [review]

**Pull request merged**